main.js

```diff
@@ -19,15 +19,19 @@ function Hoverable() {
 	var win = window;
 
 	function init() {
-		win.document.body.setAttribute('data-o-hoverable--js', '');
+		win.document.documentElement.setAttribute('data-o-hoverable--js', '');
 		touchSupport();
 	}
 
+	function updateClasses(addOrRemove) {
```

Comment:
It would break demos

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.